src/home-task-3/services/auth.ts

```diff
@@ -0,0 +1,49 @@
+import bcrypt from 'bcrypt';
+import User from '../models/User.js';
+import jwt from 'jsonwebtoken';
+
+export interface loginResponse {
+  success: boolean;
+  statusCode: number;
+  message: string;
+}
+class AuthService {
+  static async loginUser(login: string, password: string): Promise<loginResponse> {
+    const user = await User.findOne({ where: { login: login } });
+    if (user) {
+      const passwordIsValid = await bcrypt.compare(password, user.password);
+      if (passwordIsValid) {
+        const accessToken = jwt.sign(
+          {
+            id: user.id,
+            login: user.login,
+          },
+          `${process.env.ACCESS_TOKEN_SECRET}`,
+          { expiresIn: '5m' },
+        );
+
+        return {
+          success: true,
+          statusCode: 200,
+          message: accessToken,
```

Comment:
this will be quite misleading for frontend devs if you send both `access-token` and error message inside the same attribute `message`. if you have time and will, please make 4 attributes: success, statusCode, token and message